lib/exec.js

```diff
@@ -69,16 +73,30 @@ function * initFeature(context, heroku, callback) {
     })
   }
 
+  const app = yield heroku.get(`/apps/${context.app}`, {
```

Comment:
It looks like we are making the request to `/apps/:appid` twice here.. can we just move `app` into the promises and `yield` on it afterwards?

Reply:
Oh shoot, I meant to remove that call from the promises object. We have to make the `/apps/:appid` call first and then the others because the buildpacks call will fail and produce the wrong error message. I'll get that fixed.

Reply:
Fixed. Thank you.